soufi/finders/yum.py

```diff
@@ -230,7 +232,24 @@ def do_task(target, *args):
     """Run the target callable in a subprocess and return its response."""
     queue = Queue()
     process = Process(target=target, args=(queue,) + args)
-    process.start()
+    try:
+        process.start()
+    except RuntimeError as e:
+        if 'not using fork' in str(e):
+            sys.exit(
+                textwrap.dedent(
+                    """
```

Comment:
Just FYI I would accept and prefer this:
```
textwrap.dedent("""\
    wibble
    blobble
""")
```
But no biggie.

Reply:
Oh, is that how you get Black to leave that alone?  TIL...

Reply:
Well it's how I've traditionally done it but who knows with Black these days...